homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -98,15 +98,14 @@
 
     async def async_discover(discovery_payload):
         """Discover and add an MQTT alarm control panel."""
+        discovery_data = discovery_payload.__discovery_data__
```

Comment:
Why are there double underscores before/after the name ? Those are reserved for system values.

Reply:
You can add `_discovery_data_`. But why not make it `_discovery_data_`?

Reply:
Yeah, OK, changed to `discovery_data`.